docs/sphinx_guide/lists.rst

```diff
@@ -241,11 +268,11 @@ Hlists
 .. hlist::
     :columns: 2
 
-    - .. figure:: /images/yi_jing_01_chien.jpg
+    - .. figure:: https://source.unsplash.com/200x200/daily?cute+puppy
 
          This is a short caption for a figure.
 
-    - .. figure:: /images/yi_jing_01_chien.jpg
+    - .. figure:: https://source.unsplash.com/200x200/daily?cute+puppy
```

Comment:
can we keep one of these links pulling from a local image file, as we do this a lot in qiskit docs

Reply:
fixed